libs/providers/flagd/src/e2e/step-definitions/evaluation.ts

```diff
@@ -0,0 +1,364 @@
+import {
```

Comment:
I'm curious why you removed "spec" from these files.

Reply:
they are containing the test logic, but are not the actual test files, i am exporting the logic now, so i can reuse it in each provider test seperately

Reply:
Got it.